simulator/README.md

```diff
@@ -0,0 +1,19 @@
+# Graph Simulator
+
+This is a stand-alone utility that allows us to run different scenarios on the whole of social graph and test
+the desired behavior against the sdk latest implementation.
+
+## Initialized State
+Calculating the initial state for all te graphs and keys is time-consuming, and we've already calculated
```

Comment:
```suggestion
Calculating the initial state for all the graphs and keys is time-consuming, and we've already calculated
```

Reply:
nit: Also we should mention to build the sdk first, my ran picked up old binary and failed